index.html

```diff
@@ -384,9 +384,45 @@ <h2>
                     <var>message</var>)</code>.</p>
                   </li>
                   <li>
-                    <p><a>Resolve</a> <var>p</var> with <var>stream</var> and
-                    abort these steps. This invocation of {{MediaDevices/getDisplayMedia()}}
-                    is now considered to have produced a new <dfn>capture-session</dfn></p>
+                    <p>
+                      This invocation of {{MediaDevices/getDisplayMedia()}} is now
+                      considered to have produced a new <dfn>capture-session</dfn>.
+                    </p>
+                  </li>
+                  <li>
+                    If <var>controller</var> is not <code>null</code>:
+                    <ol>
+                      <li>
+                        <p>
+                          Set <var>controller</var>.{{CaptureController/[[videoTrack]]}}
+                          to <var>stream</var>'s video track.    
```

Comment:
Generally yes, but there's the technical issue of detecting when the source changes through [surfaceSwitching](https://github.com/w3c/mediacapture-screen-share/pull/225/files) and closing the window when that happens. I intend to tackle this with a follow-up PR. I think this will likely work better if we specify a handle on the track rather than the source.

Reply:
(Also, capture-actions might follow to the next source, so they might still want to be tied to the track, to facilitate "track's current source.")